examples/decoding/plot_decoding_spatio_temporal_source.py

```diff
@@ -91,14 +91,14 @@
 scores = cross_val_multiscore(time_decod, X, y, cv=7, n_jobs=1)
 
 # Plot average decoding scores of 10 splits
```

Comment:
I mean, line 93 says 10 splits were used, but line 91 shows you're using 7? Unless I misread something.